.github/workflows/prepare-release.yml

```
@@ -19,8 +19,27 @@ on:
         required: true
         description: Current version (beta, like 0.69.1). Don't include `v`.
 jobs:
+  #validate-version format
+  validate-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate version format
+        run: |
+          validate_version() {
+             if [[ ! "$1" =~ ^[1-9][0-9]*\.[1-9][0-9]*\.[1-9][0-9]*(-rcv[0-9]+)?$ ]]; then
+                 echo "Invalid $2 version format. It should be like 1.0.0 or 1.0.0-rcv0014."
+                 exit 1
+             fi
+          }
+
+          validate_version "${{ inputs.candidate-stable }}" "candidate-stable"
+          validate_version "${{ inputs.current-stable }}" "current-stable"
+          validate_version "${{ inputs.candidate-beta }}" "candidate-beta"
+          validate_version "${{ inputs.current-beta }}" "current-beta"
+        shell: bash
   # Releasing opentelemetry-collector
   prepare-release:
+    needs: validate-version
```

```suggestion
    needs: 
      - validate-version
```

I believe this needs to be a list